src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -37,6 +38,56 @@ def get(self, request, organization):
         return Response({"count": result["data"][0]["count"]})
 
 
+class OrganizationEventBaseline(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
```

Comment:
Which view is going to call this? Since `baselineValue` can be provided from the request, the comparison view? The comparison view will likely need span data for this event so it'd require a second call, should it be folded in here then? 

Alternatively if this is for a comparison list, then it should be returning the difference for each item in the list, which either means `n` calls of this or changing this to provide difference for all?

Reply:
This is for the performance transaction summary, so we should only return the id here,

But you're right, I forgot to include the baselineValue in the response so we can calculate the difference :+1: